caikit/runtime/http_server/http_server.py

```diff
@@ -497,20 +497,25 @@ async def _generator() -> pydantic_response:
                             **request_params,
                         )
                     ):
-                        yield result
+                        yield {
+                            "event": "message",
```

Comment:
Since these will be standard values, we should use an enum to avoid typos down the road

Reply:
Ah good point